clientlibs/js/src/functions/markExperimentPoint.ts

```diff
@@ -1,15 +1,11 @@
 import { Interfaces, Types } from '../identifiers';
 import fetchDataService from '../common/fetchDataService';
-import { MARKED_DECISION_POINT_STATUS } from 'upgrade_types';
+import { IExperimentAssignmentv4, MARKED_DECISION_POINT_STATUS } from 'upgrade_types';
 
 interface markData {
   userId: string;
   status: MARKED_DECISION_POINT_STATUS;
-  data: {
-    site: string;
-    assignedCondition: { conditionCode: string; experimentId?: string },
-    target?: string;
-  }
+  data: IExperimentAssignmentv4;
```

Comment:
@danoswaltCL this is important change because the structure of `getAllExperimentCondition` for simple experiment and factorial experiment is different and the change need to be reflected on Java client.